CONTRIBUTING.md

```diff
@@ -100,21 +100,24 @@ In order to run **[nimi-python](https://github.com/ni/nimi-python)** System Test
 1. On a terminal, CD to the **[nimi-python](https://github.com/ni/nimi-python)** root
    directory. Then type:
 
-         tox
+         <path to python3>/python3 -m tox -e build_test,codegen,flake8,docs,pkg
+         <path to python2>/python tox
 
-   This will
+   The first line will
 
    * For each driver
       * Generate Python bindings
       * Generate [RST documentation](http://www.sphinx-doc.org/)
       * Create installer packages
-   * Run NI-FAKE unit tests
    * Run [flake8](http://flake8.pycqa.org/)
    * Generate [HTML documentation](http://www.sphinx-doc.org/)
 
+   The second line will iterate over all python versions and run the NI-FAKE unit tests
+   for each version that is installed and registered using `tox --scan`
+
 1. To clean everything and start fresh, type:
 
-         tox -e clean
+         <path to python3>/python3 -m tox -e clean
```

Comment:
Do you think we should, instead of writing `<path to python3>` everytwhere, teach how to add it to the path?

Reply:
Some places we explicitly need python 3.x and other places we explicitly need python 2.7.

At least for now.